ui/app/styles/core/navbar.scss

```diff
@@ -146,7 +146,7 @@
     z-index: 201;
   }
 
-  &.active {
+  &.is-active {
```

Comment:
While attempting to test that the nav drawer closed when clicking a link on smaller screens I noticed that it wasn't appearing on screen in the first place. Updating the class name to match the template fixed the issue.